pandas/core/indexes/base.py

```diff
@@ -71,6 +72,7 @@
 from pandas.core.arrays import ExtensionArray
 from pandas.core.base import IndexOpsMixin, PandasObject
 import pandas.core.common as com
+import pandas.core.construction as constrn
```

Comment:
> `as constrn`

we dont do this anywhere else.  if you just need `array`,  something like `from pandas.core.construction import array as pd_array`